core/src/test/java/org/apache/iceberg/TestRemoveSnapshots.java

```diff
@@ -555,8 +575,9 @@ public void testRetainLastKeepsExpiringSnapshot() {
     }
 
     // Retain last 2 snapshots and expire older than t3
-    table
-        .expireSnapshots()
+    RemoveSnapshots removeSnapshots = (RemoveSnapshots) table.expireSnapshots();
+    removeSnapshots
+        .withIncrementalCleanup(incrementalCleanup)
```

Comment:
Why doesn't this use `removeSnapshots(Table)`?

Reply:
Oversight, fixed!